docs/timeseries.md

```diff
@@ -83,10 +83,12 @@ Content-Type: text/csv
 
 ## Request Parameters
 
+(feature)=
```

Comment:
What does this do? I see if for the other query parameters, is it formatting the headings in some way?

Reply:
this is an anchor to hyperlink to this specific subsection from other parts of the documentation

Reply:
this is an anchor to hyperlink to this specific subsection from other parts of the documentation